Dms/src/V1/Gapic/DataMigrationServiceGapicClient.php

```diff
@@ -1524,11 +1712,12 @@ public function describeConversionWorkspaceRevisions($conversionWorkspace, array
      *
      * @throws ApiException if the remote call fails
      */
-    public function describeDatabaseEntities($conversionWorkspace, array $optionalArgs = [])
+    public function describeDatabaseEntities($conversionWorkspace, $tree, array $optionalArgs = [])
```

Comment:
this is a breaking change in a GA API and will need to have a fix added to the gapic generator repository